build/mathquill.d.ts

```diff
@@ -54,6 +54,7 @@ export namespace MathQuill {
         startIndex: number;
         endIndex: number;
       };
+      cursor(): any; // TODO: Cursor interface
```

Comment:
Can you (or me) make a ticket for this so the TODO links back to something concrete we'll have to do later?